themes/mcf/pages/methodology.tsx

```diff
@@ -1,19 +1,194 @@
-import Layout from "@components/layouts/Main";
-
 import { BreadCrumbs } from "@components/breadcrumbs/Breadcrumbs";
+import Layout from "@components/layouts/Main";
+import { SiteWidth } from "@components/panels/SiteWidth";
+import { SingleCol } from "@components/panels/SingleCol";
+import { SubNav } from "@components/nav/SubNav";
+import { Heading } from "@components/typography/Heading";
 
 const Methodology = () => {
   return (
     <Layout title="Methodology" description="Find the definitions, scope and principles we use to collect and categorise the laws and policies.">
-      <section>
-        <div className="container px-4">
-          <BreadCrumbs label={"Methodology"} />
-          <div className="text-content mb-12">
-            <h1 className="my-8">Methodology</h1>
-            <h2 id="introduction">Introduction</h2>
-            <p className="italic">This page was last updated on XX XXXX XX.</p>
-          </div>
-        </div>
+      <SubNav>
+        <BreadCrumbs label={"Methodology"} />
+      </SubNav>
+      <section className="pt-8">
+        <SiteWidth>
+          <SingleCol>
+            <div>
+              <Heading level={1} extraClasses="custom-header mb-2">
+                What documents are included
+              </Heading>
+              <p className="text-content mb-12">
+                The platform contains publicly available policies and project documents (such as concept notes, full proposals, implementation
+                reports) of the four Multilateral Climate Funds.
+              </p>
+              <Heading level={2} extraClasses="custom-header">
+                Definitions and classifications that are currently in use
+              </Heading>
+              <div className="text-content mb-12">
```

Comment:
Rather than putting `text-content` onto every div we can just add it onto line 17 div once

Reply:
Have had a chat and it makes sense!